### ISSUE TITLE
Support for Google Compute Engine

### ISSUE THREAD

**Issue by timothyjlaurent** [jtriley/StarCluster#383]
It would be great to adapt this to Google's system.

They have to the minute billing so that should simplify the dynamic scaling of the cluster.

**Comment by isofer** [comment]
I'm joining on this request. Also boto has support for GCE now.

**Comment by chriswhittleston** [comment]
Would also love to see GCE support added to Starcluster

**Comment by mpf** [comment]
Agreed! I've been a happy StarCluster user, but for various reasons, would like to move to GCE.
